tests/e2e/regression/BUILD

```diff
@@ -59,7 +59,10 @@ iree_lit_test_suite(
         ] + BACKEND_TESTS,
     ),
     cfg = "//tests:lit.cfg.py",
-    tags = ["hostonly"],
+    tags = [
+        "driver=vulkan",  # Multiple driver= tags allowed. Used by sanitizer CI.
```

Comment:
OK, listed all the other drivers, which IIUC is just `driver=local-task` since the other target backends in the actual `iree-run-mlir` commands are `llvm-cpu` and `vmvx` and both correspond to that driver.